testssl.sh

```diff
@@ -1222,63 +1222,73 @@ out_row_aligned() {
 }
 
 # prints text over multiple lines, trying to make no line longer than $max_width.
-# Each line is indented with $spaces and each word in $text is printed using $print_function.
+# Each line is indented with $spaces.
 out_row_aligned_max_width() {
      local text="$1"
      local spaces="$2"
      local -i max_width="$3"
-     local print_function="$4"
      local -i i len
      local cr=$'\n'
-     local line entry first=true last=false
+     local line first=true
```

Comment:
One minor request wrt to readability: For new code please use one line per variable type in the declaration. "line" has type string, "first" is boolean.